sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSender.java

```diff
@@ -62,7 +63,7 @@
 /**
  * Handles scheduling and transmitting events through proton-j to Event Hubs service.
  */
-class ReactorSender implements AmqpSendLink, AsyncAutoCloseable {
+class ReactorSender implements AmqpSendLink, AsyncAutoCloseable, Closeable {
```

Comment:
I don't remember was it AutoCloseable or Closeable?

Reply:
this should be AutoCloseable

Reply:
Other places we are using `AutoCloseable` but here we are using  `Closeable`

Reply:
My bad. Corrected to AutoClosable